TLM/TLM/Resources/lang_fr.txt

```diff
@@ -57,36 +57,36 @@ Enable_Advanced_Vehicle_AI Activer l'IA avancée des véhicules
 Vehicles_may_enter_blocked_junctions Les véhicules peuvent entrer dans les intersections bloquées
 All_vehicles_may_ignore_lane_arrows Tous les véhicules peuvent ignorer les directions des voies
 Busses_may_ignore_lane_arrows Les bus peuvent ignorer les directions des voies
-Reckless_driving Conduite téméraire (fonctionnalité Bêta)
-TMPE_Title Traffic Manager: Edition Président
+Reckless_driving Conduite téméraire
+TMPE_Title Gestionnaire de trafic : Édition Président
 Settings_are_defined_for_each_savegame_separately Paramètres définis pour chaque sauvegarde séparément
 Simulation_accuracy Précision de la simulation (une plus haute précision réduit les performances)
 Enable_highway_specific_lane_merging/splitting_rules Activer les règles spécifiques de divergence/convergence sur les autoroutes
-Drivers_want_to_change_lanes_(only_applied_if_Advanced_AI_is_enabled): Les conducteurs veulent changer de voies: (seulement si l'IA avancée est activée)
+Drivers_want_to_change_lanes_(only_applied_if_Advanced_AI_is_enabled): Les conducteurs veulent changer de voies (seulement si l'IA avancée est activée) : 
 Maintenance Maintenance
 Very_often Très souvent
 Often Souvent
 Sometimes Parfois
 Rarely Rarement
 Very_rarely Très rarement
 Only_if_necessary Seulement si nécessaire
-Nodes_and_segments Nœuds et segments
+Nodes_and_segments Intersections et segments
 Lanes Voies
 Path_Of_Evil_(10_%) Chemin du Mal (10 %)
-Rush_Hour_(5_%) Heures de Pointe (5 %)
+Rush_Hour_(5_%) Heure de Pointe (5 %)
```

Comment:
Should it be: `Heure de Pointé` (`é` not `e`) ?

Reply:
`Heure de pointe` is correct.
 It is even a french idiom, for Rush hour !
https://fr.wikipedia.org/wiki/Heure_de_pointe